ferrocene/doc/sphinx-shared-resources/src/ferrocene_qualification/upcoming.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: MIT OR Apache-2.0
+# SPDX-FileCopyrightText: The Ferrocene Developers
+
+from docutils import nodes
+from sphinx.util.docutils import SphinxRole
+from sphinx.transforms import SphinxTransform
+from sphinx.util.logging import getLogger
+
+
+BADGE_CONTENT = "coming soon"
+
+
+class UpcomingRole(SphinxRole):
+    def run(self):
+        content = self.text
+
+        try:
+            major, minor = content.split(".")
+            role_version = (int(major), int(minor))
```

Comment:
So no point release planned?

Reply:
There will probably be point releases in the future, but the fixes included within them won't need the :upcoming: role.